code/modules/core_implant/cruciform/machinery/cruciformforge.dm

```diff
@@ -0,0 +1,147 @@
+#define LOAD "load"
+#define WORK "work"
+#define DONE "done"
+
+/obj/machinery/neotheology/cruciformforge
+	name = "Cruciform Forge"
+	desc = "This unique piece of technology can be fed biomatter, plasteel, and gold in order to print more cruciforms. Only a select few NeoTheologian architects know how to create these machines. "
+	icon = 'icons/obj/neotheology_machinery.dmi'
+	icon_state = "cruciforge"
+
+	density = TRUE
+	anchored = TRUE
+	layer = 2.8
```

Comment:
why do you use this layer?

Reply:
This layer commonly used in all machines. Parent of cruciformforge doesn't have this layer.